app/models/messages/EnRouteEvent.scala

```
@@ -92,4 +96,12 @@ object EnRouteEvent {
         }
       </ENROUEVETEV>
   }
+
+  implicit lazy val xmlReader: XmlReader[EnRouteEvent] = (
+    (xmlPath \ "PlaTEV10").read[String],
+    (xmlPath \ "CouTEV13").read[String],
+    (xmlPath \ "CTLCTL" \ "AlrInNCTCTL29").read(intReader) map (intValue => if (intValue == 1) true else false),
```

Could we move this logic to the XMLReader class to make it more reusable like the LocalDate reads, we may also want to throw a parse exception if we get anything else than a 1 or a 0

---

I like this suggestion, we can create a `XmlReader[Boolean]` that reads an int and tries to parse it as a Boolean. It would be the same logic here but would probably look something like 

```
implicit val booleanFromIntReader: XmlReader[Boolean] = intReader.map(intValue => if (intValue == 1) true else false)
```